proto/cosmos/offchain/v1alpha1/offchain.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package cosmos.offchain.v1alpha1;
+
+option go_package="github.com/cosmos/cosmos-sdk/x/offchain";
+
+import "gogoproto/gogo.proto";
+
+// MsgSignData defines an arbitrary, general-purpose, off-chain message
+message MsgSignData {
+    // signer is the sdk.AccAddress of the message signer
+    bytes signer = 1 [(gogoproto.jsontag) = "signer", (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
While bytes is for sure more "correct", we decided to use `string signer` everywhere in the SDK, to have well-formatted (bech32) addresses in txs.